README/ReleaseNotes/v616/index.md

```diff
@@ -48,6 +48,14 @@ They were deprecated before, or never ported from configure, make to CMake.
 
 ## Core Libraries
 
+### Automatic activation of implicit multithreading
+
+For the `root` binary, IMT is now automatically enabled on MacOS and Windows machines, if it was enabled during the configuration.
```

Comment:
I think we need to be more explicit and avoid any possible ambiguity with -DIMT=ON and ROOT::EnableImplicitMT().
What about
'''
Provided that the build configuration featured IMT (`-DIMT=ON`), implicit multithreading (`ROOT::EnableImplicitMT()`) is now automatically enabled at runtime on...
'''